timelock-impl/src/main/java/com/palantir/atlasdb/timelock/TimelockNamespaces.java

```diff
@@ -95,6 +102,8 @@ public static Optional<String> toUserAgent(@Nullable RequestContext context) {
      * server-side Jersey interfaces (which are just used in tests)
      */
     public TimeLockServices get(String namespace, Optional<String> userAgent) {
+        activeServicesToTime.merge(
```

Comment:
Do we need to bother with a merge over a simple put? This can lead to going back in time if there's some weird clock drift happening, but that should be insignificant given:
a) the requirements of the API and,
b) how long we wait before we consider a client to have expired

Reply:
Maybe preemptive effort at performance, but I did this to avoid synchronization when the time hasn't changed enough recently